stdpopsim/catalog/HomSap/demographic_models.py

```diff
@@ -1548,3 +1548,136 @@ def _ooa_4pop():
 
 
 _species.add_demographic_model(_ooa_4pop())
+
+
+def _ancient_europe():
+    id = "AncientEurope_4P21"
```

Comment:
The naming convention for model IDs is to use the citation's author initial in the last bit of the ID. Below, you've suggested the primary author is (will be?) Allentoft, so the ID should be AncientEurope_4A21.